redis/asyncio/connection.py

```diff
@@ -906,6 +918,12 @@ def get(self) -> SSLContext:
             context = ssl.create_default_context()
             context.check_hostname = self.check_hostname
             context.verify_mode = self.cert_reqs
+            if self.verify_flags_config:
+                for flag, enabled in self.verify_flags_config:
+                    if enabled:
+                        context.options |= flag
```

Comment:
Should we use `context.verify_flags` instead?

https://docs.python.org/3/library/ssl.html#context-creation